src/Binovo/ElkarBackupBundle/Command/RemoteRestoreCommand.php

```diff
@@ -0,0 +1,76 @@
+<?php
```

Comment:
General comments:
- I think we shouldn't use "machine". Better use "host" or "server".
- Don't write in first person (we). You won't be doing anything; elkarbakup host will do it ;)
- Take care of correct indentation
- Taker care of correct copyright notices (in new files I think it is better to put your name; also use correct year).

Functionally, it's a bit strange that user has to put in a full local path for the source of the restore (maybe I missed something sorry). Better put a link in backup navigation, so that one can ask to restore an specific file or directory without having to write it?

Don't really understand what's the goal of this really, as restoring over original backup source location is not advisable...

Reply:
Hi @elacunza 

I just remove the  <p>We suggest to configure a new host before starting this process.</p>
from the restore help panel, If anyone need to restore in a new server they would see that until they add a new one, is not going to be on the list.

About the restore remote path we should give the reasons why they souldn't restore over the original data/folder. 

 <p>You're advised not to restore over the original data. For security reasons elkarbackup does not have original files ownership.</p> 

Would you add some more adviced?

thanks!